arctic/store/_pickle_store.py

```diff
@@ -85,7 +84,7 @@ def write(self, arctic_lib, version, symbol, item, _previous_version):
             try:
                 # If it's encodeable, then ship it
                 b = bson.BSON.encode({'data': item})
-                if len(b) < _MAX_BSON_ENCODE:
+                if len(b) < MAX_BSON_ENCODE:
```

Comment:
Should we add a check that falls back to using pickle if the object size is larger than say 10MB, given this will be stored in the single version document?

Reply:
Sure